www.data.gov/energy/highlights/index.html

```diff
@@ -314,6 +314,22 @@ <h2 class="entry-title">APIs from the Energy Information Administration</h2>
               </header>
                             <article class="no-image">
                   <p>The Energy Information Administration collects statistics on energy generation, distribution and consumption in the United States. Their new APIs provide developers easy access to EIA&#8217;s extensive data on electricity, petroleum, natural gas, and more.</p>
+                  <p>
+                    Through its application programming interface (API), the U.S. Energy Information Administration (EIA) makes available free and open data to our customers about the major energy sources used in the United States. The data in the API is also available in bulk file, in Excel via the add-in, in Google Sheets via an add-on, and via widgets that embed interactive data visualizations of EIA data on any website. By making EIA data available in machine-readable formats, private, non-profit, and public customers can easily use this data to find new ways to innovate, research, and make important decisions, all powered by public data.
+
+                    Currently, our API contains the following main data sets:
+
+                    <li>Hourly electricity operating data, including actual and forecast demand, net generation, and the power flowing between electric systems</li>
+                    <li>408,000 electricity series organized into 29,000 categories</li>
+                    <li>30,000 State Energy Data System series organized into 600 categories</li>
+                    <li>115,052 petroleum series and associated categories</li>
+                    <li>34,790 U.S. crude oil imports series and associated categories</li>
+                    <li>11,989 natural gas series and associated categories</li>
+                    <li>132,331 coal series and associated categories</li>
+                    <li>3,872 Short-Term Energy Outlook series and associated categories</li>
+                    <li>368,466 Annual Energy Outlook series and associated categories</li>
+                    <li>92,836 international energy series</li>
```

Comment:
I thought `<li>` always had to be in either `<ol>` or `<ul>` for it to be good html, any reason it's not?

Reply:
good catch, nick!
https://stackoverflow.com/questions/52717274/does-li-always-need-to-be-nested-between-ul-and-ol-not-nav

Reply:
Tl;DR - yes